apps/services/routers.py

```diff
@@ -262,6 +262,30 @@ async def read_request_details(
     }
 
 
+@services_router.get("/{university_id}/return_user_document/{user_request_id}",
```

Comment:
1. don't use "_" in your url, use "-"
2. Don't use "return" word
3. See best practices about plural/singular recources. So use documentS instead of document
4. This endpoint should return all documents of user, not linked with some user request, that's why we don't need param user_request_id. Please delete it

Reply:
Here in url `user_request_id`, but in endpoint we don't use it, so it's miss-click, I will replace it